lib/view/my_crosswords.dart

```diff
@@ -214,11 +220,40 @@ class ModifyCrosswordNameScreenState extends State<ModifyCrosswordNameScreen> {
             )));
   }
 
+  void showEmptyNameAlert(BuildContext context) {
+    Widget okButton = TextButton(
+      child: const Text("OK"),
+      onPressed: () {
+        Navigator.pop(context);
+      },
+    );
+
+    AlertDialog alert = AlertDialog(
+      title: const Text("Nie udało się zapisać zmiany nazwy krzyżówki!"),
+      content: const Text(
+          "Nazwa krzyżówki nie może być pusta lub identyczna z poprzednią."),
+      actions: [okButton],
+    );
+
+    showDialog(
+      context: context,
+      builder: (BuildContext context) {
+        return alert;
+      },
+    );
+  }
+
   void saveCrosswordName(Photo photo, String text) {
     PhotoRepository photoRepository = PhotoRepository();
     photo.name = text;
     photoRepository.updatePhoto(photo);
   }
+
+  bool isCrosswordNameWrong(String oldCrosswordName, String newCrosswordName) {
+    if (oldCrosswordName == newCrosswordName) return true;
```

Comment:
I think it's unnecessary why the user can't change to the same name, he might have changed his mind or something else. Importantly, you need to check whether the name is unique among his other crosswords.